src/binfhe/lib/rgsw-acc-dm.cpp

```diff
@@ -85,7 +85,6 @@ RingGSWEvalKey RingGSWAccumulatorDM::KeyGenDM(const std::shared_ptr<RingGSWCrypt
 
     DiscreteUniformGeneratorImpl<NativeVector> dug;
     NativeInteger Q{params->GetQ()};
-    dug.SetModulus(Q);
```

Comment:
would like to make sure that dug doesn't need to be constructed with the modulus "Q".

Reply:
same